Purchases/FoundationExtensions/Data+Extensions.swift

```diff
@@ -28,6 +28,12 @@ extension NSData {
         return deviceTokenString
     }
 
+    var uuid: UUID? {
+        var bytes = [UInt8](repeating: 0, count: self.length)
+        self.getBytes(&bytes, length: self.length * MemoryLayout<UInt8>.size)
```

Comment:
does this deliver the same output as 
```swift
let bytes = [UInt8](self)
return NSUUID(uuidBytes: bytes) as UUID
```
?

Reply:
shit! the output is the same, so...KISS please 😂 let me update it 💃🏼